src/test/ui/extern/extern-crate-visibility.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unused_imports)]
```

Comment:
This should be unnecessary, UI tests allow lints from the `unused` group by default.